src/manage-apps/go-market-docs/app-center-ux-guidelines-enterprise.markdown

```diff
@@ -177,6 +179,7 @@ This section provides a summarized checklist of the required and recommended com
   * Must clearly indicate this is an integration with Concur (required)
   * Must include links to your terms and conditions and privacy policy (required)
   * Verification apps, should provide a link to the inquiry form (strongly recommended)
+  * The Partner application should not exchange the Request token for the Access token until the User has completed all of the actions on your Landing Page.
```

Comment:
Suggested text: "Your application should only exchange the request token for the access token once the user has completed all actions on your landing page (required)."

I assume this also applies to end user apps and we should add this to the "consumer" version as well?